ref/net46/Microsoft.Build.Tasks.Core/Microsoft.Build.Tasks.Core.cs

```diff
@@ -1166,7 +1166,7 @@ public partial class WinMDExp : Microsoft.Build.Tasks.ToolTaskExtension
         public bool UTF8Output { get { throw null; } set { } }
         [Microsoft.Build.Framework.RequiredAttribute]
         public string WinMDModule { get { throw null; } set { } }
-        protected internal override void AddCommandLineCommands(Microsoft.Build.Tasks.CommandLineBuilderExtension commandLine) { }
+        protected internal override void AddResponseFileCommands(Microsoft.Build.Tasks.CommandLineBuilderExtension commandLine) { }
```

Comment:
Looked on Index and nobody's extending WinMDExp, so I guess breaking this is fine :)